lib/src/package_analysis/shapes.dart

```diff
@@ -153,6 +214,23 @@ class GlobalPropertyShape extends GlobalShapeBase {
   Map<String, dynamic> toJson() => _$GlobalPropertyShapeToJson(this);
 }
 
+/// A Shape for describing a typedef.
+/// Any type parameters are ignored/omitted in the alias name.
+@sealed
+@JsonSerializable()
+class TypedefShape extends GlobalShapeBase {
+  /// The [ClassShape.id] that this typedef points to, or null if defined as function type
```

Comment:
TODO: test to see if https://dart-lang.github.io/linter/lints/comment_references.html no longer triggers here (and elsewhere in this file) now that https://github.com/Dart-Code/Dart-Code/issues/4114 is fixed

Reply:
### before
```
C:\Users\Atom\Desktop\pana>dart --version
Dart SDK version: 2.17.6 (stable) (Tue Jul 12 12:54:37 2022 +0200) on "windows_x64"

C:\Users\Atom\Desktop\pana>dart analyze .\lib\src\package_analysis
Analyzing package_analysis...

   info • shapes.dart:69:18 • Only reference in scope identifiers in doc comments. • comment_references
   info • shapes.dart:73:18 • Only reference in scope identifiers in doc comments. • comment_references
   info • shapes.dart:77:18 • Only reference in scope identifiers in doc comments. • comment_references
   info • shapes.dart:81:18 • Only reference in scope identifiers in doc comments. • comment_references
   info • shapes.dart:85:18 • Only reference in scope identifiers in doc comments. • comment_references
   info • shapes.dart:89:18 • Only reference in scope identifiers in doc comments. • comment_references
   info • shapes.dart:242:12 • Only reference in scope identifiers in doc comments. • comment_references

7 issues found.
```

### after
```
C:\Users\Atom\Desktop\pana>dart --version
Dart SDK version: 2.19.0-146.0.dev (dev) (Fri Aug 26 18:55:25 2022 -0700) on "windows_x64"

C:\Users\Atom\Desktop\pana>dart analyze .\lib\src\package_analysis
Analyzing package_analysis...
No issues found!
```